### ISSUE TITLE
[Question] Question title

### ISSUE THREAD

**Issue by MittelmanDaniel** [TransformerLensOrg/TransformerLens#766]
### Question

When I try to load llama into the hooked transformer I get a warning

```
WARNING:root:You are not using LayerNorm, so the writing weights can't be centered! Skipping
```

Is this something I should be worried about? Llama should use layer norm if I remember correctly.

This is my code:
```
import transformer_lens.utils as utils
import torch
from transformer_lens import HookedTransformer

from transformers import AutoTokenizer, LlamaForCausalLM, LlamaTokenizer

torch.set_grad_enabled(False)

device = utils.get_device()

model = HookedTransformer.from_pretrained("meta-llama/Llama-3.2-1B-Instruct", device=device)
```

**Comment by Hzfinfdu** [comment]
Hi, I am not one of the maintainers of TL but I have been working with Llama models recently. First IMO the current implementation for Llama 3.1 and 3.2 is incorrect (https://github.com/TransformerLensOrg/TransformerLens/pull/764). And it is recommended to use `from_pretrained_no_processing` to load 7B-level modules (maybe bc bfloat16 precision causing errors when processing weights).

You are encountering this warning because llama models use RMSNorm. Though they are sometimes called interchangeably with layernorm but is necessary to distinguish here since RMSNorm, unlike layernorm, does not center its input. This should not matter if you use `from_pretrained_no_processing`.

**Comment by MittelmanDaniel** [comment]
Is llama 3 affected by those issues?

Thanks for answering!

**Comment by Hzfinfdu** [comment]
I have updated the setting in Llama3, 3.1 and 3.2 models. idk how long will it take to be merged into TL. But the change is actually easy to make. Its only one-line addition. You can see https://github.com/TransformerLensOrg/TransformerLens/pull/764 for details and apply it to your local version if you want to use them asap.

**Comment by bryce13950** [comment]
The is going up to pip right now with the fix for both 3.1, and 3.2. Thanks @Hzfinfdu for implementing the fix, and answering this question!
